packages/amplication-server/src/core/outdatedVersionAlert/outdatedVersionAlert.service.ts

```diff
@@ -152,6 +170,33 @@ export class OutdatedVersionAlertService {
             latestVersion,
           },
         });
+
+        this.kafkaProducerService
+          .emitMessage(KAFKA_TOPICS.TECH_DEBT_CREATED_TOPIC, <
+            TechDebt.KafkaEvent
+          >{
+            key: {},
+            value: {
+              resourceId: service.id,
+              resourceName: service.name,
+              workspaceId: workspace.id,
+              projectId: template.projectId,
+              createdAt: Date.now(),
+              techDebtId: alert.id,
+              envBaseUrl: this.configService.get<string>(Env.CLIENT_HOST),
+              externalId: encryptString(userId),
+              resourceType: service.resourceType,
+              projectName: project.name,
+              alertType: alert.type,
+              alertInitiator: template.name,
+            },
+          })
+          .catch((error) =>
```

Comment:
move the code that raise the kafka event into a new funciton 'raiseNotifications' and call this function from the this.create() function.

this way all types of alerts will eventually raise the notifications in the same way

also, the alerts should be sent to all users in the current workspace (and not just the current user)
later, we will add logic to select specific users based on permissions and ownership